lib/formtastic/helpers/enum.rb

```diff
@@ -3,10 +3,8 @@ module Helpers
     # @private
     module Enum
       # Returns the enum (if defined) for the given method
-      def enum_for(method) # @private
-        if @object.respond_to?(:defined_enums)
-          @object.defined_enums[method.to_s]
-        end
+      def enum_for(method) # @private        
+        @object.defined_enums[method.to_s] if @object.respond_to?(:defined_enums)
```

Comment:
I don't agree with the change. In this case, two line condition is great. Not only that you can actually measure if the branch is tested by coverage, but also it is more readable I'd say.